components/signupForm.tsx

```diff
@@ -108,10 +112,12 @@ export const SignupForm = () => {
       <Space h="lg" />
 
       <Modal
-        opened={true}
+        opened={isSuccess}
         onClose={handleModalClose}
         title="Success!"
-        message="You have successfully registered. We're working on finding you a ___ and will be in touch with you as soon as possible!"
+        message={`Your registration was successful. We're working on matching you with a ${
+          userType === 'refugee' ? 'host' : 'refugee'
+        } and will be in touch with you as soon as possible!`}
```

Comment:
looks great! if the user type is a "host," I would also include a "Thank you for your support in these tough times" or something like that to show appreciation

Reply:
I really like this, good idea! Definitely going to add that before merging with main.